pyEOS/eos.py

```diff
@@ -157,17 +157,29 @@ def load_candidate_config(self, filename=None, config=None):
         else:
             self.candidate_config.load_config(config=config)
 
-    def compare_config(self):
+    def compare_replace_config(self):
         """
 
-        :return: A string showing the difference between the running_config and the candidate_config. The running_config is
-            loaded automatically just before doing the comparison so there is no neeed for you to do it.
+        :return: A string showing the difference between the running_config 
+            and the candidate_config, assuming the entire running conf will be
+            replaced by the candidate. The running_config is loaded 
+            automatically just before doing the comparison so there is no 
+            neeed for you to do it.
         """
 
         # We get the config in text format because you get better printability by parsing and using an OrderedDict
         self.load_running_config()
         return self.running_config.compare_config(self.candidate_config)
 
+    def compare_merge_config(self):
```

Comment:
I don´t like this method for two reasons:
1. compare_config should provide this once the config session feature is 100% available.
2. We are adding application logic in this library. I think pyEOS should just be a wrapper of the eAPI for python and don´t include any application logic.